css/components/guide-nav.css

```diff
@@ -24,3 +24,42 @@
   display: inline;
   font-weight: bold;
 }
+
+.lgd-guide-nav__list-item--active {
+  font-weight: bold;
+}
+
+.lgd-guide-nav__list-item > a {
+  font-weight: normal;
+}
+
+.block-localgov-guides-contents {
+  margin-top: var(--spacing-largest);
+  margin-bottom: var(--spacing-largest);
+}
+
+.lgd-prev-next__link {
+  flex-wrap: wrap;
+  padding: var(--spacing-smaller);
+  text-decoration: none;
+}
+
+.lgd-prev-next__link  > .lgd-prev-next__label {
+  font-weight: bold;
+}
+
+a.lgd-prev-next__link .lgd-prev-next__label {
+  font-weight: bold;
+}
```

Comment:
Can this all be reduced to something like:
```
.lgd-prev-next__label {
  font-weight: bold;
}
```

This looks like it's it's the same element being set to `bold` twice.